week3/homework/src/index.js

```diff
@@ -1,3 +1,50 @@
-'use strict';
+const express = require('express');
+const app = express();
 
-// TODO: Write the homework code in this file
+const bodyParser = require('body-parser');
+const getList = require('./getList');
+const getToDo = require('./getTodo');
+const deleteAll = require('./deleteAll');
+const deleteOne = require('./deleteOne');
+const addOne = require('./addToDo');
+const markDone = require('./markDone');
+const editToDo = require('./editToDo');
+
+app.use(express.json());
+app.use(bodyParser.json());
+
+app.post('/addTodo', (req, res) => {
+  const toWrite = req.body.todo.description;
+  addOne(req, res, toWrite);
+});
+
+app.get('/todos', (req, res) => {
```

Comment:
the noun is _todo_ and should be consistent

Reply:
of course I update this